jb/testing/microbenchmark_config.cpp

```diff
@@ -38,6 +38,10 @@ namespace defaults {
 #define JB_DEFAULTS_microbenchmark_size 0
 #endif
 
+#ifndef JB_DEFAULTS_microbenchmark_timeseries
```

Comment:
Please create a new configuration class in bm_time_delay_estimator_many.  This is supposed to be the configuration class for all microbenchmarks, that "timeseries" parameter would not apply to many of them.  I was just lucky in bm_time_delay_estimator.cpp because no new configuration parameters were needed.

You can look at jb/itch5/bm_order_book.cpp for an example.